ArduPlane/tiltrotor.cpp

```diff
@@ -562,7 +562,16 @@ void Tiltrotor::vectoring(void)
     } else {
         const float yaw_out = motors->get_yaw()+motors->get_yaw_ff();
         const float roll_out = motors->get_roll()+motors->get_roll_ff();
-        float yaw_range = zero_out;
+        const float yaw_range = zero_out;
+
+        // Scaling yaw with throttle
+        const float throttle = motors->get_throttle_out();
+        const float scale_min = 0.5;
+        const float scale_max = 2.0;
```

Comment:
We could change this to 1 for more conservative scaling that would not increase gains at low throttle.

Reply:
It is worth noting that these scaling factors were trailed using a lua script on a vehicle with thrust vectoring for yaw and were found to work well for a range of thottles (climbing and descending yaw manouvers in stabilise) so this scaling range does appear appropriate.